code/modules/power/lighting/light.dm

```diff
@@ -518,47 +518,47 @@
 		// create a light tube/bulb item and put it in the user's hand
 		drop_light_tube(user)
 		return
-	var/protection_amount = 0
-	var/mob/living/carbon/human/electrician = user
 
-	if(istype(electrician))
-		var/obj/item/organ/internal/stomach/maybe_stomach = electrician.getorganslot(ORGAN_SLOT_STOMACH)
+	var/protected = FALSE
+
+	if(istype(user) && !HAS_TRAIT(user, TRAIT_RESISTHEAT) && !HAS_TRAIT(user, TRAIT_RESISTHEATHANDS))
```

Comment:
wait, doesn't this mean that ethereals who don't have both of these traits won't be able to feed off electricity? do ethereals have this as an innate trait? i think it's intentional to have this check lower than the ethereal code because i think they were probably made to ignore how hot it is (mmmm yumby light)

Reply:
I guess my point is at the end, why should any of these traits prevent ethereal feeding if an ethereal somehow gets one of those traits?